bedrock/newsletter/forms.py

```diff
@@ -263,10 +264,8 @@ def __init__(self, newsletters, locale, data=None, *args, **kwargs):
             lang, country = lang.split('-', 1)
         else:
             country = ''
-            if lang_file_has_tag('mozorg/newsletters', locale, 'country_region_122019'):
-                regions.insert(0, ('', _lazy('Select country or region')))
-            else:
-                regions.insert(0, ('', _lazy('Select country')))
+            regions.insert(0, ('', ftl_lazy('newsletter-form-select-country-or-region',
```

Comment:
@pmac I'm working here on migrating our standard newsletter form to Fluent. Things are mostly working, with the exception of when I'm trying to use Fluent IDs in things like standard Django form widgets.

I'm struggling to work out  what I need to do here. Any pointers?

https://gist.github.com/alexgibson/2e7d4a7badf23ee98912dba8ecba8cf2